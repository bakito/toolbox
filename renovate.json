{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "dependencyDashboard": true,
  "customManagers": [
    {
      "customType": "regex",
      "description": "Update toolbox _VERSION variables in Makefile",
      "fileMatch": [
        "^Makefile$"
      ],
      "matchStrings": [
        "# renovate: packageName=(?<packageName>.+?)\\s+.+?_VERSION \\?= (?<currentValue>.+?)\\s"
      ],
      "datasourceTemplate": "go"
    }
  ]
}
