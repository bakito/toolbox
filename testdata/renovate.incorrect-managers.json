{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "customManagers": [
    {
      "customType": "regex",
      "datasourceTemplate": "docker",
      "description": "Update toolbox tools in Makefile",
      "fileMatch": [
        "^Docker$"
      ],
      "matchStrings": [
        "foo"
      ]
    }
  ]
}
