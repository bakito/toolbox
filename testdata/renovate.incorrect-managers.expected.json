{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "customManagers": [
    {
      "customType": "regex",
      "datasourceTemplate": "go",
      "description": "Update toolbox tools in Makefile",
      "fileMatch": [
        "^Makefile$"
      ],
      "matchStrings": [
        "# renovate: packageName=(?\u003cpackageName\u003e.+?)\\s+.+?_VERSION \\?= (?\u003ccurrentValue\u003e.+?)\\s"
      ]
    }
  ]
}
